backend/apps/readings/views.py

```diff
@@ -35,6 +25,16 @@ class DetailDocument(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = DocumentSerializer
 
 
+@api_view(['GET'])
+def document_analysis(request):
+    """
+    Init a DocumentAnalysis, and serialize it to send to the frontend.
+    """
+    queryset = Document.objects.get(pk=1)
```

Comment:
plz make a comment here that we're hardcoding this single document into this view for prototyping

Reply:
I added the comment.